ros2topic/ros2topic/verb/echo.py

```diff
@@ -201,7 +201,7 @@ def msg_to_ordereddict(msg, truncate_length=None):
     # We rely on __slots__ retaining the order of the fields in the .msg file.
     for field_name in msg.__slots__:
         value = getattr(msg, field_name, None)
-        value = _convert_value(value)
+        value = _convert_value(value, truncate_length)
```

Comment:
Since `truncate_length` is a keyword argument please pass it explicitly as such: `truncate_length=truncate_length`.

Reply:
@dirk-thomas thanks for review, updated the code.